Integrations/SplunkPy/SplunkPy.py

```diff
@@ -555,18 +555,11 @@ def splunk_parse_raw_command():
 
 def test_module(service):
     if demisto.params().get('isFetch'):
-        t = datetime.utcnow() - timedelta(days=3)
+        t = datetime.utcnow() - timedelta(hours=1)
         time = t.strftime(SPLUNK_TIME_FORMAT)
         kwargs_oneshot = {'count': 1, 'earliest_time': time}
         searchquery_oneshot = demisto.params()['fetchQuery']
-        oneshotsearch_results = service.jobs.oneshot(searchquery_oneshot, **kwargs_oneshot)  # type: ignore
-        reader = results.ResultsReader(oneshotsearch_results)
-        for item in reader:
-            if item:
-                demisto.results('ok')
```

Comment:
if the query will be invalid we get an exceptin.
so we don't need to check results.

Reply:
But the main point in this test is to validate that the search query input is valid. Don't think that we should remove it

Reply:
What will happen if query is empty, it is not mandatory field (for some reason) and has no default value. Try to run it  with empty query and see the error

Reply:
`isFetch` should be set to `False`